test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+# Missing Tests:
+#   - verification of Maximum Length Allowed in Fields
+#   - verification of Special Attribute Types
+#
+
+
+Feature: create entities requests (POST) in NGSI v2
+  As a context broker user
+  I would like to  create entities requests in NGSI v2
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @happy_path
+  Scenario:  create several entities in NGSI v2
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_happy_path  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "3" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_type    | room                    |
+      | entities_id      | room2                   |
+      | attributes_name  | random=5                |
+      | attributes_value | 017-06-17T07:21:24.238Z |
```

Comment:
Yes, in this case, I create 3 entities with 2 different attributes, but attributes values are the same in each one.